nvbench_helper/nvbench_helper/nvbench_helper.cuh

```diff
@@ -33,7 +33,17 @@ NVBENCH_DECLARE_TYPE_STRINGS(uint128_t, "U128", "uint128_t");
 
 using complex = cuda::std::complex<float>;
 
-NVBENCH_DECLARE_TYPE_STRINGS(complex, "C64", "complex");
+#if _CCCL_HAS_NVFP16()
+NVBENCH_DECLARE_TYPE_STRINGS(__half, "Half", "half");
```

Comment:
Nit: I'm late to the party on this, but I'd recommend `F16` and `BF16` to better match the other short identifiers and keep the table widths under control. Food for thought in a future PR 🙂